plugins/guests/redhat/cap/change_host_name.rb

```diff
@@ -15,16 +15,20 @@ def self.change_host_name(machine, name)
             basename = name.split('.', 2)[0]
             comm.sudo <<-EOH.gsub(/^ {14}/, '')
               # Update sysconfig
-              sed -i 's/\\(HOSTNAME=\\).*/\\1#{name}/' /etc/sysconfig/network
+              if ls /etc/sysconfig | grep network; then
```

Comment:
we can do a file check with:

`if [ -f /etc/sysconfig/network ]; then`

Reply:
Also safer as grepping on the result of `ls` would match any file with "network" in it's name inside of the folder leading to possible false positives.